src/avgScrabbleScoreFromFile.js

```diff
@@ -11,14 +11,15 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  * sample-words.txt.
  */
 function avgScrabbleScoreFromFile(fileName) {
-  // This is your job. :)
+  let words = fs.readFileSync(fileName, 'utf-8').split("\n");
+  let avg = avgScrabbleScore(words);
+  let score = avg.toFixed(2);
```

Comment:
1. Why truncate to two digits?
2. `toFixed` returns a string (you can't truncate floats, so the only way you can guarantee two digits in JS is to use a string), which means the function no longer works as intended. For example, you couldn't use the return value of this function numerically.

Reply:
I truncated because I was trying to avoid a long decimal. I will leave it as is, definitely don't want it to return a string.